### DISCUSSION TITLE
Data uncertainty

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#47] [people; relating]
Data after all is one number; more important is assumed perception structure to interpret the number.
- Forrester: data is one realization of reality
- Rubin and Gelman's imputation

<img width="954" alt="image" src="https://user-images.githubusercontent.com/30194633/194708778-7475ba74-2829-4f51-a5bd-b9d6aed4f5f3.png">

**Comment by tomfid** [comment]
The hard part is knowing what "all possible realizations" really look like. If you have five industrial companies, or five large cities, at least you can compare experiences. There's only one planet though.
